### PR TITLE
Add my account to Top Menu, only show when user is allowed to maintai…

### PR DESCRIPTION
**Pull request by asig2016**
…n his own data

### CODE REVIEW

**Comment by ralfbecker** [comment]
@birgit-egroupware, @Stefan-U: What do you think, should we add "My account" unconditionally, if the user has rights to change some of his contact-data? 
If he has no rights (default) it is not shown, neither if he is an admin on such a system.
Ralf

File: api/src/Framework.php [hunk 1]
```diff
@@ -1363,6 +1367,18 @@ protected function add_preferences_topmenu($type='prefs')
 					));
 				}
 				break;
+            case 'useraccount':
+                $config = Config::read('phpgwapi');
+                if ($config['own_account_acl'] && is_array($config['own_account_acl']) && count($config['own_account_acl'])>0){
```
Comment (ralfbecker) [inline api/src/Framework.php:1372]: This should be: if (!empty($config['own_account_acl']) &&
To not raise a PHP Warning

File: api/src/Framework.php [hunk 2]
```diff
@@ -1307,6 +1307,10 @@ protected function add_preferences_topmenu($type='prefs')
 				'title' => 'Access',
 				'hook'  => 'acl_rights',
 			),
+            'useraccount' => array(
```
Comment (ralfbecker) [inline api/src/Framework.php:1310]: Please fix the indention to use tab.

**Comment by birgit-egroupware** [comment]
Hi Ralf, I support that request.Birgit Birgit BeckerEGroupware GmbHLeibnizstraße 17D-67663 KaiserslauternEMail ***@***.*** 0631 / 31657-13Telefax 0631 / 3165726Mobil    0170 / 3147833Skype birgit-klGeschäftsbereiche EGroupware & EGroupware Forum, digital ROCK     Geschäftsführung Ralf und Birgit BeckerRegistergericht Kaiserslautern HRB 3587
-------- Ursprüngliche Nachricht --------Von: Ralf Becker ***@***.***> Datum: 29.01.24  08:42  (GMT+01:00) An: EGroupware/egroupware ***@***.***> Cc: Birgit Becker ***@***.***>, Mention ***@***.***> Betreff: Re: [EGroupware/egroupware] Add my account to Top Menu, only show when user is allowed to maintai… (PR #141) 
@birgit-egroupware, @Stefan-U: What do you think, should we add "My account" unconditionally, if the user has rights to change some of his contact-data?
If he has no rights (default) it is not shown, neither if he is an admin on such a system.
Ralf

File: api/src/Framework.php (hunk 1 above)
Comment (asig2016) [reply api/src/Framework.php]: Done!

File: api/src/Framework.php (hunk 2 above)
Comment (asig2016) [reply api/src/Framework.php]: Fixed!

**Pull request merged**